app-wear-shared/shared/src/main/java/info/nightscout/rx/weardata/EventData.kt

```diff
@@ -13,13 +14,20 @@ sealed class EventData : Event() {
 
     fun serialize() = Json.encodeToString(serializer(), this)
 
+    fun serializeByte() = ProtoBuf.encodeToByteArray(serializer(), this)
     companion object {
 
         fun deserialize(json: String) = try {
             Json.decodeFromString(serializer(), json)
         } catch (ignored: Exception) {
             Error(System.currentTimeMillis())
         }
+
+        fun deserializeByte(byteArray: ByteArray) = try {
+            ProtoBuf.decodeFromByteArray(serializer(), byteArray)
```

Comment:
Same, gives warning during build

Reply:
Yes the warning arrived one or 2 days ago when I updated Android Studio and my working branch with latest dev 😥...
Itried to fix it with suggested proposal, but watch face was no more sent to the watch (I think we have to manually accept something somewhere...)
I couldn't use standard json serialuzer (Size of json serialization is between 3 and 4 times greater, so about 1,7mb for my test watchface, butthe threshold to be able to send data to watch is at about 1mB...
With protobuf serializer, size is reduced to about 400/500kb and can be sent (as bytearray) to the watch...
I will try to see if I can fix it (with an authorization request, or by using another binary serialyser...)